tests/test_vidispine_api.py

```diff
@@ -255,6 +255,8 @@ def get_hex(self):
                     'Content-Type': 'application/octet-stream',
                     'Accept': 'application/xml'
                 }
+
+                size_to_send = int(str(testfilesize))
```

Comment:
why do we need to convert to string then back to int? what is testfilesize?

Reply:
Because if you use int as is it prints an L on the end sometimes. The int conversion it to make sure it goes in to the call as an int. I know it looks silly, but this does work.

Reply:
testfilesize is the name of the object that was holding the number to be tested.

Reply:
the L simply means Long, and it does not matter to the comparison if it's there or not - take this out please https://docs.python.org/2/library/stdtypes.html#numeric-types-int-float-long-complex (second paragraph) https://www.python.org/dev/peps/pep-0237/

Reply:
apparently the L suffix is added by the long integer's __repr__ method, so if you use .format instead of str() when compiling the headers it should go away

Reply:
It looks like the L I was seeing may not have been a problem. However, I have made a change to the code here which now uses format to remove a problem if there is one.